docs/usage/faq-troubleshooting.md

```diff
@@ -1,9 +1,31 @@
 # Troubleshooting and FAQs
 
-## I get a warning during centrifuge_kreport process with exit status 255
+## I get a warning during CENTRIFUGE_KREPORT or KRAKENTOOLS_COMBINEKREPORTS_CENTRIFUGE process with exit status 255
 
 When a sample has insufficient hits for abundance estimation, the resulting `report.txt` file will be empty.
 
-When trying to convert this to a kraken-style report, the conversion tool will exit with a status code `255`, and provide a `WARN`.
+When trying to convert this to a kraken-style report or merging together, the tools will exit with a status code `255`, and provide a `WARN`.
 
-This is _not_ an error nor a failure of the pipeline, just your sample has no hits to the provided database when using centrifuge.
+This is _not_ an error nor a failure of the pipeline, just your sample has no hits to the provided database when using Centrifuge.
+
+## Why does any error or failed process from BRACKEN_BRACKEN get ignored?
+
+If Kraken2 doesn't classify any reads (100% unclassified) Bracken will fail on that input.
+Having no reads taxonomically classified can be a reasonable outcome for some samples, and we don't want to treat this as an error.
+
+Therefore to allow the remainder of a given run to complete if a single Bracken task fails, [we set the Nextflow `errorStrategy` to `ignore`](https://github.com/nf-core/taxprofiler/blob/5d3ee5513a84f92773c8376c55b5f4da39835307/conf/base.config#L61-L63).
+You will still get a warning in the Nextflow console output and log that a Bracken task failed.
+
+If you want to change this behaviour, you can override the `errorStrategy` in your own Nextflow configuration file.
+
+For example:
+
+```nextflow
+process {
+    withName: BRACKEN_BRACKEN {
+        errorStrategy = 'retry'
+    }
+}
+```
+
+Other `errorStrategy` options can be found on the [Nextflow documentation](https://www.nextflow.io/docs/latest/process.html#errorstrategy).
```

Comment:
Wow, I just learned that I can set the error strategy for some of my meta-val processes. Never thought about it before.